src/main/flight/imu.c

```diff
@@ -344,46 +359,11 @@ STATIC_UNIT_TESTED void imuUpdateEulerAngles(void) {
 static void imuCalculateEstimatedAttitude(timeUs_t currentTimeUs)
 {
     static timeUs_t previousIMUUpdateTime;
-
-    bool useAcc = false;
-    bool useMag = false;
-    bool useCOG = false;
-    float courseOverGround = 0;
-
     const timeDelta_t deltaT = currentTimeUs - previousIMUUpdateTime;
     previousIMUUpdateTime = currentTimeUs;
 
-#ifdef USE_MAG
-    if (sensors(SENSOR_MAG)) {
-        useMag = true;
-    }
-#endif
-#if defined(USE_GPS)
-    if (sensors(SENSOR_GPS) && STATE(GPS_FIX) && gpsSol.numSat >= 5 && gpsSol.groundSpeed >= 600) {
-        static bool hasInitializedGPSHeading = false;
-        // In case of a fixed-wing aircraft we can use GPS course over ground to correct heading
-        if(STATE(FIXED_WING)) {
-            courseOverGround = DECIDEGREES_TO_RADIANS(gpsSol.groundCourse);
-            useCOG = true;
-        } else {
-            float tiltDirection = atan2_approx(attitude.values.roll, attitude.values.pitch); // For applying correction to heading based on craft tilt in 2d space
-            courseOverGround = tiltDirection + DECIDEGREES_TO_RADIANS(gpsSol.groundCourse);
-
-            if (!hasInitializedGPSHeading && GPS_distanceToHome > 50) { // Initially correct the gps heading, we can deal with gradual corrections later
-                attitude.values.yaw = RADIANS_TO_DECIDEGREES(courseOverGround);
-                hasInitializedGPSHeading = true;
-            }
-
-            useCOG = true; // Tell the IMU to correct attitude.values.yaw with this data
-        }
-    }
-#endif
-
 #if defined(SIMULATOR_BUILD) && defined(SKIP_IMU_CALC)
     UNUSED(imuMahonyAHRSupdate);
-    UNUSED(useAcc);
-    UNUSED(useMag);
-    UNUSED(useCOG);
     UNUSED(courseOverGround);
```

Comment:
courseOverGround is also throwing errors on some targets. Probably some ifdef missing to isolate the feature.